nbs/models.dilated_rnn.ipynb

```diff
@@ -375,7 +375,11 @@
     "    `hist_exog_list`: str list, historic exogenous columns.<br>\n",
     "    `stat_exog_list`: str list, static exogenous columns.<br>\n",
     "    `loss`: PyTorch module, instantiated train loss class from [losses collection](https://nixtla.github.io/neuralforecast/losses.pytorch.html).<br>\n",
-    "    `learning_rate`: float=1e-3, initial optimization learning rate (0,1).<br>\n",
+    "    `max_steps`: int, maximum number of training steps.<br>\n",
+    "    `learning_rate`: float, Learning rate between (0, 1).<br>\n",
```

Comment:
Can we add the default values to the docstrings?

Reply:
Done